CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 5.3.4 (Next)
```

Comment:
Let's bump the version to 5.4.0 part of this PR for semver.

Reply:
Sure. I put today's date in place of `Next` and guessed the next version would be under the heading `5.4.1`.